### PR TITLE
Babylon tanzimat rework

### PR DESCRIPTION
**Pull request by Unintuition-Time**
Ottoman Tanzimat Rework; Pending final changes post-1.5 Patch

### CODE REVIEW

File: common/country_definitions/kurdistan_(merge_with_china_rework_definitions_file).txt
Comment (HiddeLekanne) [inline common/country_definitions/kurdistan_(merge_with_china_rework_definitions_file).txt:1]: don't rename files like that lol

File: common/history/countries/bul - bulgaria.txt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+COUNTRIES = {
+	c:BUL = {
+		effect_starting_technology_tier_4_tech = yes
+		
+		add_technology_researched = academia
+		add_technology_researched = law_enforcement
+		add_technology_researched = mandatory_service
+		add_technology_researched = urban_planning
+		add_technology_researched = line_infantry
+		set_tax_level = medium
```
Comment (HiddeLekanne) [inline common/history/countries/bul - bulgaria.txt]: consistent place for tax level please (after tech and law)

File: common/history/countries/kur - kurdistan.txt [hunk 2]
```diff
@@ -0,0 +1,25 @@
+COUNTRIES = {
+	c:KUR = {
+		ig:ig_landowners = {
+			add_ruling_interest_group = yes
+		}
```
Comment (HiddeLekanne) [inline common/history/countries/kur - kurdistan.txt]: consistent place for ruling class please (after set tax level)

File: common/journal_entries/babylon_bulgaria_je.txt [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+	icon = "gfx/interface/icons/event_icons/event_newspaper.dds"
+	
+	complete = {
+		AND = {
+			has_technology_researched = nationalism
+			NOT = {
+			is_subject_of = c:TUR
+			}
+		}
+	}
```
Comment (Danarca) [inline common/journal_entries/babylon_bulgaria_je.txt]: `complete = { }` is `AND `statements per default ;)

File: common/history/pops/01_south_europe.txt
Comment (HiddeLekanne) [inline common/history/pops/01_south_europe.txt:1]: what changed?

File: common/history/pops/08_middle_east.txt
Comment (HiddeLekanne) [inline common/history/pops/08_middle_east.txt:1]: what changed?

File: events/babylon_bulgarian_independence.txt [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+			ruler = yes
+			culture = cu:bulgarian
+			interest_group = ig_armed_forces
+			traits = {
+			}
+		}
+		create_character = {
+			heir = yes
+			culture = cu:bulgarian
+			birth_date = 1836.1.1
```
Comment (Danarca) [inline events/babylon_bulgarian_independence.txt]: Is the birth_date necessary? If this pops late you may end up with an heir older than the ruler :s

File: events/sick_man_events.txt [hunk 5]
```diff
@@ -0,0 +1,664 @@
...
+			value = 30
+		}
+		change_relations = {
+			country = c:GBR
+			value = 30
+		}
+		change_relations = {
+			country = c:PRU
+			value = 30
+		}
```
Comment (Danarca) [inline events/sick_man_events.txt]: How about NGF/Germany?

File: common/country_definitions/kurdistan_(merge_with_china_rework_definitions_file).txt
Comment (Danarca) [inline common/country_definitions/kurdistan_(merge_with_china_rework_definitions_file).txt:1]: You can just call it kurdistan.txt, if I'm correct Emifas will get an option to sort out a merge conflict when he gets that far :)

File: common/production_methods/08_monuments.txt [hunk 6]
```diff
@@ -6,7 +6,7 @@ pm_default_building_eiffel_tower = {
 
 	country_modifiers = {
 		workforce_scaled = {
-			country_prestige_add = 100
+			country_prestige_add = 150
```
Comment (Danarca) [inline common/production_methods/08_monuments.txt]: A bit random that you're doing monuments as part of the Ottomans, but okay :p

**Review by Danarca — CHANGES_REQUESTED** [review]
Besides what I've pointed out you should probably also give the files a bit of auto-formatting. Would help with readability.

File: common/journal_entries/00_sick_man.txt [hunk 7]
```diff
@@ -0,0 +1,1007 @@
...
+		trigger_event = {
+			id = sick_man.3
+		}
+	}
+	current_value = {
+		value = var:sick_man_var
+	}
+	on_monthly_pulse = {
+		random_events = {
+			10 = sick_man.4
```
Comment (HiddeLekanne) [inline common/journal_entries/00_sick_man.txt]: this is a 100% chance to get this even, this is not random xd.

File: common/journal_entries/00_sick_man.txt [hunk 8]
```diff
@@ -0,0 +1,1007 @@
...
+		}
+	}
+	invalid = {
+		custom_tooltip = {
+			text = finished_sick_man_tt
+			has_variable = sick_man_complete
+		}
+	}
+	transferable = no
+	weight = 5000
```
Comment (HiddeLekanne) [inline common/journal_entries/00_sick_man.txt:88]: weights determine order, if you want a fixed order do not make them all 5000 but one 5000 and another 5001

File: common/production_methods/08_monuments.txt (hunk 6 above)
Comment (Unintuition-Time) [reply common/production_methods/08_monuments.txt]: Funny enough it actually mattered because changing hagia sophia's prestige count significantly determined whether or not they were a GP. I got it to make them right on the line to becoming one through these changed

File: events/sick_man_events.txt (hunk 5 above)
Comment (Unintuition-Time) [reply events/sick_man_events.txt]: Would that be an or statement? Or an and?

File: events/babylon_bulgarian_independence.txt (hunk 4 above)
Comment (Unintuition-Time) [reply events/babylon_bulgarian_independence.txt]: These were ported from emifas' branch, sorry. Same with the other bulgaria factor

File: events/sick_man_events.txt (hunk 5 above)
Comment (Danarca) [reply events/sick_man_events.txt]: For the sake of readability, I'd do an `if` statement, with an `exists = c:PRU` in the `limit`, for example. Then put the `change_relations` for the specified country in there.

File: common/production_methods/08_monuments.txt (hunk 6 above)
Comment (Danarca) [reply common/production_methods/08_monuments.txt]: 😂 Fair enough. But seems like this can easily change over the course of a campaign, so it _may_ be the wrong thing to adjust..

File: events/babylon_bulgarian_independence.txt (hunk 4 above)
Comment (Danarca) [reply events/babylon_bulgarian_independence.txt]: Sounds like we'll be having the same points to make when Emifas is done then, whoever grabs that pull request.

File: common/journal_entries/00_sick_man.txt [hunk 9]
```diff
@@ -0,0 +1,1007 @@
...
+			}
+		}
+	}
+	transferable = no
+	weight = 5000
+	progressbar = yes
+}
+
+# Nationalism
+je_sick_man_separatism = {
```
Comment (HiddeLekanne) [inline common/journal_entries/00_sick_man.txt]: just make them two different journal entries lol. The constant use if = { limit = { has_variable = absolute_tanzimat}} makes stuff less readable.

File: common/journal_entries/00_sick_man.txt [hunk 10]
```diff
@@ -0,0 +1,1007 @@
...
+			has_variable = sick_man_complete
+		}
+	}
+	transferable = no
+	weight = 5000
+	progressbar = yes
+}
+
+# Literacy and Universities
+je_sick_man_education = {
```
Comment (HiddeLekanne) [inline common/journal_entries/00_sick_man.txt]: just make them two different journal entries lol. The constant use if = { limit = { has_variable = absolute_tanzimat}} makes stuff less readable.

File: common/journal_entries/00_sick_man.txt [hunk 11]
```diff
@@ -0,0 +1,1007 @@
...
+}
+
+# Secularism
+je_sick_man_liberal_secularism = {
+	icon = "gfx/interface/icons/event_icons/event_scales.dds"
+	complete = {
+		scope:journal_entry = {
+			is_goal_complete = yes
+		}
+		has_law = law_type:law_guaranteed_liberties
```
Comment (HiddeLekanne) [inline common/journal_entries/00_sick_man.txt:750]: That's pretty harsh, isn't right of assembly enough?

File: common/journal_entries/00_sick_man.txt [hunk 12]
```diff
@@ -0,0 +1,1007 @@
...
+				state_region = s:STATE_MONTENEGRO
+			}
+			any_scope_state = {
+				state_region = s:STATE_DOBRUDJA
+			}
+			any_subject_or_below = {
+				any_scope_state = {
+					state_region = s:STATE_MOLDAVIA
+				}
+			}
```
Comment (HiddeLekanne) [inline common/journal_entries/00_sick_man.txt:781]: These are partial ownership btw.

File: common/journal_entries/00_sick_man.txt [hunk 13]
```diff
@@ -0,0 +1,1007 @@
...
+		owns_entire_state_region = STATE_ALEPPO
+		owns_entire_state_region = STATE_SYRIA
+		owns_entire_state_region = STATE_LEBANON
+		owns_entire_state_region = STATE_PALESTINE
+		owns_entire_state_region = STATE_TRANSJORDAN
+		owns_entire_state_region = STATE_ADANA
+		OR = {
+			has_law = law_type:law_agrarianism
+			has_law = law_type:law_interventionism
+		}
```
Comment (HiddeLekanne) [inline common/journal_entries/00_sick_man.txt:927]: Why are there law requirements here?

File: common/journal_entries/00_sick_man.txt [hunk 14]
```diff
@@ -0,0 +1,1007 @@
...
+	}
+	invalid = {
+		custom_tooltip = {
+			text = finished_sick_man_tt
+			has_variable = sick_man_complete
+		}
+	}
+	on_weekly_pulse = {
+		events = {
+			egyptian_crisis_events.1
```
Comment (HiddeLekanne) [inline common/journal_entries/00_sick_man.txt:983]: Why is this in weekly_pulse? do you want to fire the event every week?

File: common/journal_entries/00_sick_man.txt [hunk 15]
```diff
@@ -0,0 +1,1007 @@
...
+						owns_entire_state_region = STATE_LOWER_EGYPT
+					}
+				}
+			}
+			custom_tooltip = je_sick_man_advance
+			change_variable = {
+				name = sick_man_var
+				add = 1
+			}
+		}
```
Comment (HiddeLekanne) [inline common/journal_entries/00_sick_man.txt]: Why is this here when there is already a journal entry?

File: common/journal_entries/00_sick_man.txt [hunk 16]
```diff
@@ -0,0 +1,1007 @@
...
+	}
+	complete = {
+		scope:journal_entry = {
+			is_goal_complete = yes
+		}
+		AND = {
+			has_law = law_type:law_state_religion
+			has_law = law_type:law_inquisition
+			has_law = law_type:law_no_womens_rights
+			has_law = law_type:law_tenant_farmers
```
Comment (HiddeLekanne) [inline common/journal_entries/00_sick_man.txt:1052]: ehm why not a not serfdom?

File: common/journal_entries/00_sick_man.txt [hunk 17]
```diff
@@ -0,0 +1,1007 @@
...
+		set_variable = {
+			name = sick_man_absolute_tradition_var
+			value = 0
+		}
+	}
+	complete = {
+		scope:journal_entry = {
+			is_goal_complete = yes
+		}
+		AND = {
```
Comment (HiddeLekanne) [inline common/journal_entries/00_sick_man.txt:1048]: Why are we checking again in complete, when the progressbar already does the same check to fill up?

File: common/modifiers/babylon_modifiers.txt [hunk 18]
```diff
@@ -448,3 +448,272 @@ babylon_greener_grass_modifier = {
...
+	icon = gfx/interface/icons/timed_modifier_icons/modifier_flag_positive.dds
+	state_radicals_from_discrimination_mult = -0.10
+}
+
+balkan_absolute_peace = {
+	icon = gfx/interface/icons/timed_modifier_icons/modifier_flag_positive.dds
+	country_decree_cost_mult = -0.10
+}
+
+sick_man_liberal_education = {
```
Comment (HiddeLekanne) [inline common/modifiers/babylon_modifiers.txt:554]: no education access?

File: common/modifiers/babylon_modifiers.txt
Comment (HiddeLekanne) [inline common/modifiers/babylon_modifiers.txt:1]: ehm... aren't you overwriting vanilla modifiers here? If so, can you keep them in vanilla files?

File: common/production_methods/08_monuments.txt (hunk 6 above)
Comment (HiddeLekanne) [reply common/production_methods/08_monuments.txt]: I don't like these monument changes as this will further put nations without monuments behind. Also the Eiffel tower can't be build until midgame? So why are we changing it?

File: events/babylon_bulgarian_independence.txt (hunk 4 above)
Comment (HiddeLekanne) [reply events/babylon_bulgarian_independence.txt]: So tag him, @Emifas

File: events/sick_man_events.txt (hunk 5 above)
Comment (HiddeLekanne) [reply events/sick_man_events.txt]: can't they just use  change_relations ?= {

File: events/sick_man_events.txt [hunk 19]
```diff
@@ -0,0 +1,664 @@
...
+		set_variable = {
+			name = sick_man_separatist_var
+			value = 0
+		}
+		add_journal_entry = {
+			type = je_sick_man_main
+		}
+	}
+	option = {
```
Comment (HiddeLekanne) [inline events/sick_man_events.txt:30]: not default?

File: events/sick_man_events.txt [hunk 20]
```diff
@@ -0,0 +1,664 @@
...
+			}
+			every_subject_or_below = {
+				limit = {
+					NOT = {
+						any_overlord_or_above = {
+							NOT = {
+								ROOT = this
+							}
+						}
+					}
```
Comment (HiddeLekanne) [inline events/sick_man_events.txt:308]: Very complicated way to write that it needs to be a direct subject lol.

File: events/sick_man_events.txt [hunk 21]
```diff
@@ -0,0 +1,664 @@
...
+		remove_modifier = sick_man_of_europe
+		remove_modifier = outmoded_bureaucracy
+		remove_modifier = cizye_tax
+		remove_modifier = stagnant_economy
+		remove_modifier = janissary_legacy
+		remove_modifier = rampant_piracy
+		remove_modifier = kurdish_question
+		remove_modifier = balkan_disarray
+		remove_modifier = imbalanced_education
+		remove_modifier = internal_strife
```
Comment (HiddeLekanne) [inline events/sick_man_events.txt]: I believe you should use ?= since the modifier can already be removed. Also I believe ?= works, but I'm not 100% sure. Otherwise it's if = { limit = { modifier_exists }} for every modifier.

**Review by HiddeLekanne — CHANGES_REQUESTED** [review]

File: events/sick_man_events.txt (hunk 21 above)
Comment (Danarca) [reply events/sick_man_events.txt]: `remove_modifier ?= modifier_name` definitely works and should be used here

File: common/history/buildings/01_south_europe.txt
Comment (HiddeLekanne) [inline common/history/buildings/01_south_europe.txt:1]: If you did not copy these directly from vanilla, then you need to do that, and make your changes again. Otherwise this file will include our naval changes (which we removed from babylon core)

File: common/history/buildings/08_middle_east.txt
Comment (HiddeLekanne) [inline common/history/buildings/08_middle_east.txt:1]: If you did not copy these directly from vanilla, then you need to do that, and make your changes again. Otherwise this file will include our naval changes (which we removed from babylon core)

File: common/country_definitions/kurdistan_(merge_with_china_rework_definitions_file).txt
Comment (Unintuition-Time) [reply common/country_definitions/kurdistan_(merge_with_china_rework_definitions_file).txt:1]: I didn't name this, apologies.

File: common/history/pops/01_south_europe.txt
Comment (Unintuition-Time) [reply common/history/pops/01_south_europe.txt:1]: Changed Bulgaria and Dobrudja to be owned by the Vilayet

File: common/history/pops/08_middle_east.txt
Comment (Unintuition-Time) [reply common/history/pops/08_middle_east.txt:1]: Changed the relevant states and provinces to be owned by Kurdistan

File: common/journal_entries/00_sick_man.txt (hunk 9 above)
Comment (Unintuition-Time) [reply common/journal_entries/00_sick_man.txt]: That would mean triggers relevant to the JE in vanilla would need to be entirely overhauled to account for both JEs. @Danarca recommended this as a simpler resolution to the issue.

File: common/journal_entries/00_sick_man.txt (hunk 12 above)
Comment (Unintuition-Time) [reply common/journal_entries/00_sick_man.txt:781]: Intentional. All three states start as partially owned.

File: common/journal_entries/00_sick_man.txt (hunk 13 above)
Comment (Unintuition-Time) [reply common/journal_entries/00_sick_man.txt:927]: It's about reforming the Vilayets, which require the Syrian Vilayet to be returned to the Empire. I can work to make that more clear to the player.

File: common/journal_entries/00_sick_man.txt (hunk 15 above)
Comment (Unintuition-Time) [reply common/journal_entries/00_sick_man.txt]: What do you mean? This was Vanilla.

File: common/journal_entries/00_sick_man.txt (hunk 14 above)
Comment (Unintuition-Time) [reply common/journal_entries/00_sick_man.txt:983]: What do you mean? This was Vanilla.

File: common/journal_entries/00_sick_man.txt (hunk 15 above)
Comment (Unintuition-Time) [reply common/journal_entries/00_sick_man.txt]: Oh I see the issue here. I wasn't sure how to handle the code for this given that it should be granting you claims to do the Egyptian entry more easily. Original Vanilla design was to gate the Egyptian Entry until after completion of this one but I gave the player more flexibility.

File: common/journal_entries/00_sick_man.txt (hunk 16 above)
Comment (Unintuition-Time) [reply common/journal_entries/00_sick_man.txt:1052]: They shouldn't be reforming to more progressive laws.

File: common/journal_entries/00_sick_man.txt (hunk 17 above)
Comment (Unintuition-Time) [reply common/journal_entries/00_sick_man.txt:1048]: My understand was that this helped it show up properly in the JE, but if I'm mistaken I could remove it.

File: common/modifiers/babylon_modifiers.txt (hunk 18 above)
Comment (Unintuition-Time) [reply common/modifiers/babylon_modifiers.txt:554]: Just to make them have a better boost? I suppose I could I just didn't think it necessary as the main goal was to make the other law categories easier to pass by strengthening liberal pops.

File: common/production_methods/08_monuments.txt (hunk 6 above)
Comment (Unintuition-Time) [reply common/production_methods/08_monuments.txt]: I can revert them, I think the only caveat before doing so is it'll guarantee the Ottomans can't be a GP until they complete at least a few of the reforms so idk if that is good or bad for their original placement

File: events/babylon_bulgarian_independence.txt (hunk 4 above)
Comment (Emifas) [reply events/babylon_bulgarian_independence.txt]: Hi, if I make the heir have an undefined birth date he tends to be the same age as the auto-generated ruler. Stems from the issue with Bulgaria under the Ottomans being a monarchy and so the historical ruler at game start will generate an heir which also needs to be removed. I'm unsure if there is a way to kill the heir so i've used that as a workaround

File: common/country_definitions/kurdistan_(merge_with_china_rework_definitions_file).txt
Comment (Emifas) [reply common/country_definitions/kurdistan_(merge_with_china_rework_definitions_file).txt:1]: oh that was me lmao, i'll rename the file so i can just resolve the merge conflict when it happens.

Comment (Emifas) [reply common/country_definitions/kurdistan_(merge_with_china_rework_definitions_file).txt:1]: i was unsure about how github dealt with file conflicts so i gave it a funny name. i'll rename the file so i can just resolve the merge conflict when it happens later on.

File: common/history/pops/01_south_europe.txt
Comment (Emifas) [reply common/history/pops/01_south_europe.txt:1]: the pops were moved from the ottoman states of bulgaria and dobrudja (which no longer exist due to the ownership change) to the bulgarian/danubian vilayet states.

File: common/history/pops/08_middle_east.txt
Comment (Emifas) [reply common/history/pops/08_middle_east.txt:1]: the pops were also assigned to kurdistan

File: common/journal_entries/babylon_bulgaria_je.txt (hunk 3 above)
Comment (Emifas) [reply common/journal_entries/babylon_bulgaria_je.txt]: fixed

File: common/history/countries/bul - bulgaria.txt (hunk 1 above)
Comment (Emifas) [reply common/history/countries/bul - bulgaria.txt]: done

File: common/history/countries/kur - kurdistan.txt (hunk 2 above)
Comment (Emifas) [reply common/history/countries/kur - kurdistan.txt]: done

File: common/journal_entries/00_sick_man.txt (hunk 14 above)
Comment (HiddeLekanne) [reply common/journal_entries/00_sick_man.txt:983]: well it's silly vanilla. They do a weekly pulse for and event that is only supposed to fire once.

File: events/babylon_bulgarian_independence.txt (hunk 4 above)
Comment (HiddeLekanne) [reply events/babylon_bulgarian_independence.txt]: So why not make the heir older?

File: common/journal_entries/00_sick_man.txt (hunk 9 above)
Comment (Danarca) [reply common/journal_entries/00_sick_man.txt]: We can do 2 JEs, want help with that @Unintuition-Time ?

File: events/babylon_bulgarian_independence.txt (hunk 4 above)
Comment (Danarca) [reply events/babylon_bulgarian_independence.txt]: There's also `disinherit_character = yes` used in the scope of the heir.

File: common/journal_entries/00_sick_man.txt (hunk 15 above)
Comment (HiddeLekanne) [reply common/journal_entries/00_sick_man.txt]: So remove it out of this journal entry or you risk giving double points.

File: common/production_methods/08_monuments.txt (hunk 6 above)
Comment (HiddeLekanne) [reply common/production_methods/08_monuments.txt]: Why not decaying prestige modifier at the start of the game? through the journal entry.

File: events/babylon_bulgarian_independence.txt (hunk 4 above)
Comment (Emifas) [reply events/babylon_bulgarian_independence.txt]: used danarca's suggestion to get rid of the heir.

File: events/sick_man_events.txt (hunk 5 above)
Comment (Unintuition-Time) [reply events/sick_man_events.txt]: After some testing I am struggling to properly fulfill this without errors.

File: common/production_methods/08_monuments.txt (hunk 6 above)
Comment (Unintuition-Time) [reply common/production_methods/08_monuments.txt]: Yea that sounds good

Comment (Unintuition-Time) [reply common/production_methods/08_monuments.txt]: Actually all things considered, I think its appropriate to let it remain as is without the monuments so i'll proceed with just reverting.

File: common/journal_entries/00_sick_man.txt (hunk 7 above)
Comment (Unintuition-Time) [reply common/journal_entries/00_sick_man.txt]: What do i change it to then?

File: common/journal_entries/00_sick_man.txt (hunk 9 above)
Comment (Unintuition-Time) [reply common/journal_entries/00_sick_man.txt]: Yes please, that would be a lot of work for me to try and do on my own.

File: common/journal_entries/00_sick_man.txt (hunk 11 above)
Comment (Unintuition-Time) [reply common/journal_entries/00_sick_man.txt:750]: There is a separate tanzimat already requiring right of assembly. This one should be on par in difficulty with the Egypt tanzimat for Absolutism as these are the two options which remove internal strife which is the most intractable modifier. since the only other real requiirement is to pass total separation, this should be feasible given everything.

File: common/journal_entries/00_sick_man.txt (hunk 15 above)
Comment (Unintuition-Time) [reply common/journal_entries/00_sick_man.txt]: I think I fixed it now.

File: common/modifiers/babylon_modifiers.txt
Comment (Unintuition-Time) [reply common/modifiers/babylon_modifiers.txt:1]: Which files? this isn't attached to anything.

File: common/journal_entries/00_sick_man.txt (hunk 14 above)
Comment (Unintuition-Time) [reply common/journal_entries/00_sick_man.txt:983]: idk what you want me to do about that

File: events/sick_man_events.txt (hunk 19 above)
Comment (Unintuition-Time) [reply events/sick_man_events.txt:30]: Doesn't seem like it needs it since it's only the one entry?

Comment (Unintuition-Time) [reply events/sick_man_events.txt:30]: one option*

File: events/sick_man_events.txt (hunk 20 above)
Comment (Unintuition-Time) [reply events/sick_man_events.txt:308]: This is also a paradox moment. Let me know if you have a better way to clean it up

File: events/sick_man_events.txt (hunk 21 above)
Comment (Unintuition-Time) [reply events/sick_man_events.txt]: what is ?=

Comment (Danarca) [reply events/sick_man_events.txt]: It basically means "if this exists, do this". In this case, it might seem redundant, to do a "if `sick_man_of_europe` exists, remove it", but Vic3 starts spitting out errors if it gets the order to remove a modifier that does not exist.

File: events/sick_man_events.txt (hunk 20 above)
Comment (Danarca) [reply events/sick_man_events.txt:308]: `is_direct_subject_of = c:TUR` would work.

File: events/sick_man_events.txt (hunk 5 above)
Comment (Danarca) [reply events/sick_man_events.txt]: I can help out :)

File: common/journal_entries/00_sick_man.txt (hunk 7 above)
Comment (Danarca) [reply common/journal_entries/00_sick_man.txt]: IIRC, this is the "half-way done with Tanzimats" event, where you choose your direction, yeah..?
If so, I'd replace the `random_events` with `events`, and scrap the `10 = ` part, so it's just the event.
Then, you make sure the triggers are set up correctly so it'll only pop up at the right time, and only once.

File: common/journal_entries/00_sick_man.txt (hunk 14 above)
Comment (Danarca) [reply common/journal_entries/00_sick_man.txt:983]: Just checked this out, and it's the event that fires when Ottomans attack Egypt. We _could_ move it to a diplomatic_play on_action instead, and rewrite some of the event triggers?

**Review by HiddeLekanne — APPROVED** [review]

**Review by Danarca — APPROVED** [review]
